rest/quotes/quotes_test.go

```diff
@@ -56,8 +57,8 @@ func TestListQuotes(t *testing.T) {
 	"request_id": "a47d1beb8c11b6ae897ab76cdbbf35a3",
 	"next_url": "https://api.polygon.io/v3/quotes/AAPL?cursor=YWN0aXZlPXRydWUmZGF0ZT0yMDIxLTA0LTI1JmxpbWl0PTEmb3JkZXI9YXNjJnBhZ2VfbWFya2VyPUElN0M5YWRjMjY0ZTgyM2E1ZjBiOGUyNDc5YmZiOGE1YmYwNDVkYzU0YjgwMDcyMWE2YmI1ZjBjMjQwMjU4MjFmNGZiJnNvcnQ9dGlja2Vy",
 	"results": [
-		` + quote1 + `,
-		` + quote2 + `
+` + indent(true, quote1, "\t\t") + `,
```

Comment:
i realized we weren't indenting these correctly while updating the snapshot tests so i wrote a simple helper. it's just copied in the various test files for now